src/main/java/core/basesyntax/io/read/impl/csv/CsvReaderImpl.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.io.read.impl.csv;
+
+import core.basesyntax.io.read.ReportReader;
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class CsvReaderImpl implements ReportReader {
+    private final String reportPath;
+
+    public CsvReaderImpl(String reportPath) {
+        this.reportPath = reportPath;
+    }
+
+    public String readReport() throws FileNotFoundException {
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(reportPath))) {
+            if (reportPath.isEmpty()) {
```

Comment:
The check for an empty reportPath should be outside of the try block. You can move this check into the constructor or into a separate validation method that is called before attempting to open the file.